.github/workflows/ci-large.yml

```diff
@@ -139,7 +139,7 @@ jobs:
         v8: [11.4.183.23]
         include:
           - label: old_nginx
-            os: [ubuntu-latest]
+            os: [ubuntu-latest, macos-latest]
```

Comment:
Hmm iirc we can't have array values in `include`?

Reply:
Hm, I think you are right. I'll update the commit accordingly.